react/tic-tac-toe/src/app/screens/Game/components/Board/styles.module.scss

```diff
@@ -1,9 +1,33 @@
+@import 'src/scss/colors.scss';
+
 .board-row:after {
   clear: both;
-  content: "";
+  content: '';
   display: table;
 }
 
 .status {
   margin-bottom: 10px;
 }
+
+$square-background: $white;
```

Comment:
variables on top of the scss file please

Reply:
Sorry for that...